app/views/simulations/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @simulation.simulation_standings.size == 0 %>
+    <p>Simulating still. Come back later!</p>
+<% else %>
+    <h2><%= @simulation.name %> (<%= @simulation.runs %> runs)</h2>
+    <table>
+        <thead>
+            <tr>
+                <th>Team</th>
+                <th>Champion</th>
+                <th>Promotion</th>
+                <th>Relegation</th>
+            </tr>
+        </thead>
+        <tbody>
+            <% @simulation.simulation_standings.order("champion DESC, promotion DESC, relegation").each do |standing| %>
+                <tr>
+                    <td><%= standing.team.name %></td>
+                    <td><%= standing.champion %></td>
+                    <td><%= standing.promotion %></td>
+                    <td><%= standing.relegation %></td>
+                </tr>
+            <% end %>
```

Comment:
If there are hundreds of simulations, this will render all on one page. This may be an opportunity to [explore some gems that help with pagination in Rails](https://github.com/markets/awesome-ruby#pagination). 

I've used `kaminari` and `will_paginate` before.  I think kaminari is most popular right now, but I could be wrong.

Reply:
Created #12 to address it. I also need pagination in matches.